### DISCUSSION TITLE
How to get only those fields before save which are dehydrate(false)?

### DISCUSSION THREAD

**Discussion by zeeshantariq08** [filamentphp/filament#14885] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?
@leandrocfe 

Forms\Components\RichEditor::make($field->name)
                        ->label(ucwords(str_replace('_', ' ', $field->name)))
                        ->placeholder("Enter {$field->name}")
                        ->dehydrated(false)

**Comment by leandrocfe** [comment]
I think you can't, because they are dehydrated(false).

What is the goal?

**Reply by zeeshantariq08** [reply]
these fields needs to be saved on different table.

**Reply by leandrocfe** [reply]
[customize the creation process](https://filamentphp.com/docs/3.x/panels/resources/creating-records#customizing-the-creation-process)

**Reply by zeeshantariq08** [reply]
fields are dynamic i don't know what will be the field name in the data

**Reply by leandrocfe** [reply]
try to explain what you need and the goal
